requirements/base.in

```diff
@@ -1,4 +1,5 @@
 open-api-framework
+git+https://github.com/maykinmedia/commonground-api-common.git@issue/44-migration-error
```

Comment:
I think this can be removed and you can bump it by running `bin/compile_dependencies.sh -P commonground-api-common==2.1.0`

Reply:
Done